src/objects/_width-container.scss

```
@@ -5,15 +5,37 @@
   // On mobile, add half width gutters
   margin: 0 $govuk-gutter-half;
 
+  // Add safe area (e.g. iPhone X, Google Pixel 3)
+  @supports (margin: unquote("max(calc(0px))")) {
+    // Use safe area or mobile margin, whichever is larger
+    // Escaped due to Sass max() vs. CSS native max()
+    margin-right: unquote("max(#{$govuk-gutter-half}, calc(#{$govuk-gutter-half} + env(safe-area-inset-right)))");
+    margin-left: unquote("max(#{$govuk-gutter-half}, calc(#{$govuk-gutter-half} + env(safe-area-inset-left)))");
+  }
+
   // On tablet, add full width gutters
   @include govuk-media-query($from: tablet) {
     margin: 0 $govuk-gutter;
+
+    // Add safe area (e.g. iPhone X, Google Pixel 3)
```

AFAIK neither of the devices listed here would actually match this because it's at the tablet breakpoint – should the comment be clearer about this?

```suggestion
    // Respect 'safe area' on tablet devices to avoid placing content in e.g. notches and rounded corners
```

---

Yeah, appreciate this is a hypothetical "future" notched tablet! Just in case.

Might have one arrive some day.

These devices might be long forgotten in future so shall I change all the comments if you prefer? Thanks

---

Using a new comment for all notch @supports queries now:

```
// Respect 'display cutout' safe area (avoids notches and rounded corners)
```